HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift

```diff
@@ -828,14 +837,54 @@ extension PaymentReviewViewController {
 }
 
 extension PaymentReviewViewController: PaymentReviewViewModelDelegate {
```

Comment:
[Suggestion]: the vc is getting very big let's refactor it as 1st step extract the extension in separate files